src/api/routes/authRouter.js

```diff
@@ -7,13 +7,24 @@ import Auth from '../../middleware/auth';
 import dropToken from '../../middleware/droppedToken';
 
 const authRouter = Router();
+
+const {
+  RequestPasswordReset,
+  ConfirmPasswordReset,
+  ApplyPasswordReset,
+  register,
+  verifyAccount,
+  SignOut
+} = authController;
 const { usernameExists, emailExists } = userValidation;
-const { register, verifyAccount, SignOut } = authController;
 const { verifyToken } = Auth;
 
 authRouter.get('/signout', verifyToken, dropToken, SignOut);
 
+
 authRouter.post('/signup', validateBody('signup'), validateGender, usernameExists, emailExists, register);
 authRouter.get('/verify', verifyAccount);
-
+authRouter.post('/reset', RequestPasswordReset);
+authRouter.get('/reset/:token', ConfirmPasswordReset);
```

Comment:
Could you consider using query parameters instead? The way you have used is good, but it could be better if you make it like ```/reset/?token=sfghsgfjsfj```